openstack-cloud/src/main/resources/index.jelly

```diff
@@ -1,3 +1,3 @@
 <div>
-    Allows Jenkins to build using Cloud Servers via JClouds.
-</div>
\ No newline at end of file
+    Allows Jenkins to build using Cloud Servers in OpenStack via jclouds.
```

Comment:
Is the fact we use jclouds still relevant after we dropped support for all other providers?